src/modules/Announcement/Events/presentation/components/AddEvent.tsx

```
@@ -66,68 +62,114 @@ export function AddEventModal({ open, onOpenChange }: AddEventModalProps) {
 
-    const payload: EventRequest = {
-      name,
-      event_date: combinedDateTime,
-      image_url: imageUrl || "",
-      description,
-      venue,
-    };
+  const handleSave = async () => {
+    try {
+      if (!name.trim() || !description.trim() || !date.trim()) {
+        useToast("error", "Please fill in all required fields");
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)